libimobiledevice/2018-09-03-fb71aeef10488ed7b0e60a1c8a553193301428c0.patch

```diff
@@ -38,6 +39,60 @@ index 054e2a1..e8f9120 100644
  AM_LDFLAGS = $(libusbmuxd_LIBS) $(libplist_LIBS) ${libpthread_LIBS}
  
  noinst_LTLIBRARIES = libinternalcommon.la
+diff --git a/common/userpref.c b/common/userpref.c
+index be745cb..9e91680 100644
+--- a/common/userpref.c
++++ b/common/userpref.c
+@@ -347,6 +347,10 @@ static int X509_add_ext_helper(X509 *cert, int nid, char *value)
+ 	X509_EXTENSION *ex;
+ 	X509V3_CTX ctx;
+ 
++#ifdef HAVE_WOLFSSL
++	memset(&ctx, 0, sizeof(ctx));
++#endif
++
+ 	/* No configuration database */
+ 	X509V3_set_ctx_nodb(&ctx);
+ 
+@@ -411,7 +415,12 @@ userpref_error_t pair_record_generate_keys_and_certs(plist_t pair_record, key_da
+ 	{
+ 		/* set serial number */
+ 		ASN1_INTEGER* sn = ASN1_INTEGER_new();
++#ifdef HAVE_WOLFSSL
++		/* wolfSSL doesn't permit using 0 for serial numbers. */
```

Comment:
Let's add a reference to `RFC 5280 - 4.1.2.2`. Just to show that its not anything missing in wolfSSL.